go/kbnm/findbin.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/kardianos/osext"
+)
+
+var errKeybaseNotFound = errors.New("failed to find the keybase binary")
+
+// findKeybaseBinary returns the path to the Keybase binary, if it finds it.
+func findKeybaseBinary() (string, error) {
+	// Is it near the kbnm binary?
+	dir, err := osext.ExecutableFolder()
+	if err == nil {
+		path := filepath.Join(dir, keybaseBinary)
+		if _, err := os.Stat(path); !os.IsNotExist(err) {
```

Comment:
Last time I linked against libkb, it pushed the `kbnm` binary size from 2mb to ~28mb. Probably not worth it for a tiny bit of sugar. I suppose I could copypasta it, but also doesn't feel worth it for just two cases.